src/elements/pearl-chain/pearl-chain-leg/pearl-chain-leg.visual.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { html } from 'lit';
+
+import { describeViewports, visualDiffDefault } from '../../core/testing/private.js';
+
+import './pearl-chain-leg.js';
+
+describe('sbb-pearl-chain-leg', () => {
```

Comment:
Should we also test combinations of props? like past and disruption and so on... so we ensure everyhing looks like expected.